src/main/java/dev/slimevr/config/ConfigManager.java

```diff
@@ -0,0 +1,91 @@
+package dev.slimevr.config;
+
+import io.eiren.util.ann.ThreadSafe;
+import io.eiren.yaml.YamlException;
+import io.eiren.yaml.YamlFile;
+
+import java.io.*;
+
+
+public class ConfigManager {
+
+	private static String CONFIG_PATH = "vrconfig.yml";
+
+	private final YamlFile config = new YamlFile();
+
+	private final TrackersConfig trackersConfig;
+
+	private final FilteringConfig filteringConfig;
+
+	private final AutoboneConfig autoboneConfig;
+
+	private final WindowConfig windowConfig;
+
+	private final KeybindingsConfig keybindingsConfig;
+
+	private final BridgeConfig bridgeConfig;
+
+
+	public ConfigManager() {
+		this.trackersConfig = new TrackersConfig(this);
+		this.filteringConfig = new FilteringConfig(this);
+		this.autoboneConfig = new AutoboneConfig(this);
+		this.windowConfig = new WindowConfig(this);
+		this.keybindingsConfig = new KeybindingsConfig(this);
+		this.bridgeConfig = new BridgeConfig(this);
+	}
+
+	public void loadConfig() {
+		try {
+			config.load(new FileInputStream(new File(CONFIG_PATH)));
+		} catch (FileNotFoundException e) {
+			// Config file didn't exist, is not an error
+		} catch (YamlException e) {
+			e.printStackTrace();
+		}
+		this.trackersConfig.loadConfig();
+		this.autoboneConfig.loadConfig();
+		this.filteringConfig.loadConfig();
+		this.windowConfig.loadConfig();
+		this.bridgeConfig.loadConfig();
+	}
+
+	@ThreadSafe
+	public synchronized void saveConfig() {
+		this.trackersConfig.saveConfig();
```

Comment:
Why is this saving the tracker config? If there's a good reason then maybe it can be added as a comment, then this can be marked as resolved

Reply:
https://github.com/SlimeVR/SlimeVR-Server/pull/227/files#diff-26e9670613ebf70e6487d7401ad4cf394b1a64fe102f0f0520f59ce8084b5a29L209-L232

This is what the previous saveConfig function used to do. So i did the same to keep the same execution